### PR TITLE
Rules and schemas update

### PR DESCRIPTION
**Pull request by kprzerwa**

### CODE REVIEW

**Comment by ludmilamarian** [comment]
This pull request **introduces 2 alerts** when merging 551a2bfd83f73dd336e7dad54f7738605818dbde into 921bc991b4759ef5442586373dc9fadf2e483d99 - [view on LGTM.com](https://lgtm.com/projects/g/CERNDocumentServer/cds-dojson/rev/pr-b1a62776fb45af31ce8db95c3099340f374f1e83)

**new alerts:**

* 2 for Unused import

---

This project has automated code review enabled, but doesn't use the <a href="https://github.com/apps/lgtm-com">LGTM GitHub App</a>. Migrate over by <a href="https://github.com/apps/lgtm-com/installations/new">installing the app</a>. Read about the <a href="https://lgtm.com/blog/announcing_github_apps_migration">benefits of migrating to GitHub Apps</a> in the blog.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: cds_dojson/marc21/fields/rdm/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""CDS RDM fields."""
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/rdm/__init__.py]: wow wow wow RDM! :)

File: cds_dojson/marc21/fields/rdm/base.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+from dateutil import parser
+from dojson.errors import IgnoreKey
+
+from cds_dojson.marc21.fields.books.errors import ManualMigrationRequired
+from cds_dojson.marc21.fields.utils import clean_val, out_strip
+from cds_dojson.marc21.models.base import model
+
+
+@model.over('preprint_date', '^269__')     # item, RDM?!
```
Comment (ntarocco) [inline cds_dojson/marc21/fields/rdm/base.py]: Not sure the comment is very clear!

Comment (kprzerwa) [reply cds_dojson/marc21/fields/rdm/base.py]: I think it was a moment of weakness when I was trying to clean the schemas and fields :)

**Pull request merged**